src/plugins/data/public/ui/query_string_input/__snapshots__/query_string_input.test.tsx.snap

```diff
@@ -320,8 +320,6 @@ exports[`QueryStringInput Should disable autoFocus on EuiFieldText when disableA
               "patch": [MockFunction],
               "post": [MockFunction],
               "put": [MockFunction],
-              "removeAllInterceptors": [MockFunction],
-              "stop": [MockFunction],
```

Comment:
ping @elastic/kibana-app-arch We need to update this snapshot everytime we perform changes on client-side core APIs. Would it be possible for you to fix that test's snapshot expectations?

Reply:
AFAIK they have an issue https://github.com/elastic/kibana/issues/52181